src/tests/test_raft_repl_dev.cpp

```diff
@@ -202,7 +206,10 @@ class TestReplicatedDB : public homestore::ReplDevListener {
         g_helper->runner().execute().get();
     }
 
-    uint64_t db_num_writes() const { return m_num_commits.load(std::memory_order_relaxed); }
+    uint64_t db_commit_count() const {
+        std::shared_lock lk(db_mtx_);
+        return commit_count_;
```

Comment:
Is there any difference between `commit_count_` and `m_num_commits`?

Reply:
You are right, it is redundant, will remove that.